packages/renderer/src/api/Renderer.ts

```diff
@@ -0,0 +1,10 @@
+import Component from './Component';
+import ComponentsMap from './ComponentsMap';
+
+export default interface Renderer {
+  randomPrefix: string;
+
+  renderLayouts(componentsMap: ComponentsMap): void;
```

Comment:
There is no need to have componentsMap in arguments, as it is in the workspace configuration the Renderer service can get it from workspace directly.

Reply:
Renderer is an extension of CapsulaHub, so you have 2 variables that will be available : CAPSULAHUB_WORKSPACE and CAPSULAHUB_CONFIG (names may not be final). You need to declare them somewhere in your service.
So anywhere in your implementation you can call for example `CAPSULAHUB_WORKSPACE.components()` to get a component map.